File: 05_functional_programming.js
let languages = [
	"Java",
	"Javascript",
	"Python",
	"Rust",
	"Go"
]

let new_list = languages
	.filter(language => language.includes("J"))
	.map(language => `${language} 🍄`)

console.log(new_list)
// ['Java 🍄', 'Javascript 🍄']

File: 05_functional_programming.py
languages = [
	"Java",
	"Javascript",
	"Python",
	"Rust",
	"Go"
]

new_list = filter(lambda language: "J" in language, languages)
new_list = map(lambda language: f"{language} 🍄", new_list)

print(list(new_list))
# ['Java 🍄', 'Javascript 🍄']